CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.3.1 - 2023-01-24
```

Comment:
i think we should do 0.4.0 because this change is a new feature, not a bug fix

Reply:
fair, it was just so small it felt wrong but you are right